### PR TITLE
Update README to new GOV.UK standard

### PR DESCRIPTION
**Pull request by bradleywright**
The list of formats is probably incomplete, but it's a start.

I expect the instructions for running the app to be the most helpful bit of this documentation.

### CODE REVIEW

**Comment by dsingleton** [comment]
LGTM, huge improvement. :+1:

File: README.md [hunk 1]
```diff
@@ -1,23 +1,88 @@
...
+which uses the production
+[content API](https://github.com/alphagov/govuk_content_api) and a local copy of
+static.
+
+To run in a full development stack (with DNS, all apps running etc.) just use
+`./startup.sh`.
+
+### Running the test suite
+
+`bundle exec rake test` runs the test suite.
```
Comment (alext) [inline README.md]: This could be just `bundle exec rake`.  That's pretty standard for all ruby projects.

**Comment by bradleywright** [comment]
@alext updated and force pushed. Good to merge?

**Comment by alext** [comment]
Looks good to me.

**Pull request merged**